biogeophys/FatesPlantRespPhotosynthMod.F90

```diff
@@ -251,8 +265,24 @@ subroutine FatesPlantRespPhotosynthDrive (nsites, sites,bc_in,bc_out,dtime)
          slatop    => prt_params%slatop , &  ! specific leaf area at top of canopy, 
                                              ! projected area basis [m^2/gC]
          woody     => prt_params%woody,   &  ! Is vegetation woody or not? 
-         stomatal_intercept   => EDPftvarcon_inst%stomatal_intercept ) !Unstressed minimum stomatal conductance
-
+         stomatal_intercept   => EDPftvarcon_inst%stomatal_intercept, &  !Unstressed minimum stomatal conductance
+         flnr        => EDPftvarcon_inst%flnr, &       ! Input: fraction of leaf N in the Rubisco enzyme (gN Rubisco / gN leaf)
+         vcmax_np1   => EDPftvarcon_inst%vcmax_np1, &  !vcmax~np relationship coefficient
+         vcmax_np2   => EDPftvarcon_inst%vcmax_np2, &  !vcmax~np relationship coefficient
+         vcmax_np3   => EDPftvarcon_inst%vcmax_np3, &  !vcmax~np relationship coefficient
+         vcmax_np4   => EDPftvarcon_inst%vcmax_np4 )   !vcmax~np relationship coefficient
+
+         ! vcmax25 parameters from Bonan et al (2011) JGR, 116, doi:10.1029/2010JG001593
```

Comment:
It will be cleaner if fnr, act25, jmax_np parameters are all moved to parameter file, just like vcmax_np parameters. It will also benefit parameter tuning later.